### PR TITLE
Setting default env options for mono llvm-jit

### PR DESCRIPTION
**Pull request by fanyang-mono**

### CODE REVIEW

File: src/BenchmarksDriver/Program.cs [hunk 1]
```diff
@@ -1082,11 +1082,21 @@ public static int Main(string[] args)
...
+                    string mono_runtime_value = monoOption.Value();
+                    Console.WriteLine(mono_runtime_value);
+                    string env_value = "";
+                    if (!string.IsNullOrEmpty(mono_runtime_value) && mono_runtime_value.Contains("llvm"))
+                        env_value = "--llvm --server --gc=sgen --gc-params=mode=throughput";
+                    else
+                        env_value = "--server --gc=sgen --gc-params=mode=throughput";
+
+                    string warn_msg_part1 = "WARNING: mono runtime is used, but no value for MONO_ENV_OPTIONS was set, using defaults: '";
+                    string warn_msg_part2 = "'";
```
Comment (sebastienros) [inline src/BenchmarksDriver/Program.cs:1094]: This is never used

File: src/BenchmarksDriver/Program.cs [hunk 2]
```diff
@@ -1082,11 +1082,21 @@ public static int Main(string[] args)
 
                 if (monoOption.HasValue() && !serverJob.EnvironmentVariables.ContainsKey("MONO_ENV_OPTIONS"))
                 {
+                    string mono_runtime_value = monoOption.Value();
+                    Console.WriteLine(mono_runtime_value);
```
Comment (sebastienros) [inline src/BenchmarksDriver/Program.cs:1086]: Rogue debug string

File: src/BenchmarksDriver/Program.cs [hunk 3]
```diff
@@ -1082,11 +1082,21 @@ public static int Main(string[] args)
...
+                    string env_value = "";
+                    if (!string.IsNullOrEmpty(mono_runtime_value) && mono_runtime_value.Contains("llvm"))
```
Comment (sebastienros) [inline src/BenchmarksDriver/Program.cs:1088]: use curly braces on `if` statements

File: src/BenchmarksDriver/Program.cs (hunk 1 above)
Comment (fanyang-mono) [reply src/BenchmarksDriver/Program.cs:1094]: It is used inside `Log`

File: src/BenchmarksDriver/Program.cs (hunk 2 above)
Comment (fanyang-mono) [reply src/BenchmarksDriver/Program.cs:1086]: Oops... good catch!

File: src/BenchmarksDriver/Program.cs [hunk 4]
```diff
@@ -1082,11 +1082,24 @@ public static int Main(string[] args)
...
+                    else
+                    {
+                        env_value = "--server --gc=sgen --gc-params=mode=throughput";
+                    }
+
+                    string warn_msg_part1 = "WARNING: mono runtime is used, but no value for MONO_ENV_OPTIONS was set, using defaults: '";
+                    string warn_msg_part2 = "'";
                     Console.ForegroundColor = ConsoleColor.DarkYellow;
-                    Log("WARNING: mono runtime is used, but no value for MONO_ENV_OPTIONS was set, using defaults: '--server --gc=sgen --gc-params=mode=throughput'");
+                    Log(warn_msg_part1 + env_value + warn_msg_part2);
```
Comment (sebastienros) [inline src/BenchmarksDriver/Program.cs:1099]:
```
Log($"WARNING: mono runtime is used, but no value for MONO_ENV_OPTIONS was set, using defaults: '{env_value }'");
```

File: src/BenchmarksDriver/Program.cs [hunk 5]
```diff
@@ -1082,11 +1082,24 @@ public static int Main(string[] args)
 
                 if (monoOption.HasValue() && !serverJob.EnvironmentVariables.ContainsKey("MONO_ENV_OPTIONS"))
                 {
+                    string mono_runtime_value = monoOption.Value();
```
Comment (sebastienros) [inline src/BenchmarksDriver/Program.cs:1085]: Please follow the naming conventions already used:
`monoRuntimeValue` for instance (camel case)

**Pull request merged**